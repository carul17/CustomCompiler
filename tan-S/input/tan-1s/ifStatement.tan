main {
	var a := 5;
	var b := 6;
	var c  := true;
	if(a > b) {
		a := 7;
	}
	else {
		a := 10;
	}
	print a \n;
}