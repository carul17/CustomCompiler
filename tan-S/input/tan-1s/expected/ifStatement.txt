[ 129 ]
[ 5 133 ]
[ 10 ]
10
